applications/vanilla/views/discussion/embed.php

```diff
@@ -38,6 +39,7 @@
         if ($this->Pager->lastPage()) {
             $LastCommentID = $this->addDefinition('LastCommentID');
             if (!$LastCommentID || $this->Data['Discussion']->LastCommentID > $LastCommentID)
+                $this->Data['Discussion'] = (is_array($this->Data['Discussion'])) ? (object)$this->Data['Discussion'] : $this->Data['Discussion'];
```

Comment:
Do you remember why this was added to the PR? The same question applies for the other 2 new seemingly unused variables.

Reply:
https://github.com/vanilla/vanilla/blob/20a36a47f1e4103b6bc1b95af20d6a654c9581b3/applications/vanilla/views/discussion/embed.php#L43